### ISSUE TITLE
Notifications: No email notifications for comment replies/likes

### ISSUE THREAD

**Issue by xpurichan** [Automattic/wp-calypso#28223]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/me/notifications/comments
2. Check boxes to receive email notifications when someone likes or replies to _my comments_.

#### What I expected
We expect to receive emails when someone likes or replies to our comments. 

#### What happened instead
Instead we only receive emails when someone likes or replies to comments on **blogs we also follow**.

We can't receive email notifications when someone likes or replies to comments on a blog that we don't follow. 

I also noted that if we opt-in for browser notifications on likes or replies to comments we do receive notifications regardless of whether we follow the blogs or not. This problem is only seen with the email options.

Ideally we can provide true email support for this feature, but if we can't I wonder if we can change the wording in Calypso to make this more clear or associated with followed blog only.

#### Context / Source
#user-report

Problem was reported on 1519860-zen.

**Comment by lancewillett** [comment]
Thanks for reporting this; I am marking as "enhancement" because this could be by design or working as intended (but maybe not ideal for everyone).

Needs further investigation to confirm the behavior, then we can move ahead with what the best user experience would be (or a wording change).

**Comment by xpurichan** [comment]
This seems to have been fixed, or it simply isn't a problem with Atomic sites. I performed a test for an unrelated problem today. In this scenario I used an email address to follow comments on a test site. I had already been following this site, and all notifications were working well.

I tested unfollowing this site but kept my subscription active for comments on a single post and continued to receive notifications.
